### ISSUE TITLE
Exclude admin from Global search

### ISSUE THREAD

**Issue by SlimenTN** [sonata-project/SonataAdminBundle#5988]
Sorry for this but I had to ask, because I can't find an answer anywhere
and this guy asked this question years ago and nobody answered
https://stackoverflow.com/questions/54125748/sonata-admin-bundle-exclude-custom-admin-from-global-search

I have an admin without entity and I want to exclude it from global search to avoid `Class not found exception`
I've added `global_search="false"` inside `tag` but it didn't work:
```
<service id="campaign.calendar" class="CampaignBundle\Admin\CalendarAdmin">
      <tag name="sonata.admin" manager_type="orm" group="admin" label="Calendar" global_search="false"/>

      <argument />
      <argument />
      <argument>CampaignBundle\Controller\CalendarController</argument>
</service>
```

**Comment by core23** [comment]
Sorry, but we try to keep github issues for feature requests and bugs only. 

- Please have a look at the [sonata docs](https://sonata-project.org/bundles/)
- Use [StackOverflow](http://stackoverflow.com/questions/tagged/sonata) for general support and help others as well
- Join #sonata on [the symfony-devs slack](https://symfony.com/slack-invite) for immediate help

**Comment by SlimenTN** [comment]
but no one is answering there !

**Comment by VincentLanglet** [comment]
> but no one is answering there !

And no one will answer here.

**Comment by SlimenTN** [comment]
in the documentation they said set `global_search="false"` but it's not working !
where should look for an answer then ?

**Comment by VincentLanglet** [comment]
> in the documentation they said set `global_search="false"` but it's not working !

That's not what the documentation is saying: https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/search.html

> where should look for an answer then ?

> * Use [StackOverflow](http://stackoverflow.com/questions/tagged/sonata) for general support and help others as well
> * Join #sonata on [the symfony-devs slack](https://symfony.com/slack-invite) for immediate

**Comment by SlimenTN** [comment]
https://stackoverflow.com/questions/54125748/sonata-admin-bundle-exclude-custom-admin-from-global-search
this question was posted years ago and nobody answered
if I open another one they will mark it as duplicated
tell more about getting help from stackoverflow .

**Comment by phansys** [comment]
Feature addressed at #6609.
